### DISCUSSION TITLE
[Feature Request]: 名称识别对于罗马数字识别不准。

### DISCUSSION THREAD

**Discussion by xwndo** [jxxghp/MoviePilot#3361] [讨论]
### 当前程序版本

1.2.3

### 功能改进

比如这个：  警察故事2.Fight.Against.Evil.II.2023.2160p.WEB-DL.HEVC.HDR.AAC-HDVWEB.mp4
回识别为：  Fight Against Evil Ii

希望加强对于 II  III  IV  这类罗马数字的识别

**Comment by cng10001** [comment]
> 希望加强对于 II III IV 这类罗马数字的识别

楼主只说到罗马数字234，没发现其实罗马数字1也有问题嘛！
最近订阅封神，发有罗马数字问题，我也群里问来着~
![269607933-08c37878-3924-44ff-a746-bad67ee00400](https://github.com/jxxghp/MoviePilot/assets/115156545/8a037703-c5c8-4929-8467-7d575c5d6163)
![269608502-135fef8c-4435-4cde-bbfd-de0ae462b6bc](https://github.com/jxxghp/MoviePilot/assets/115156545/f6acd109-5b37-47d5-8107-11f097899c81)
![269608519-46da30fe-d0b0-4390-b759-955828537728](https://github.com/jxxghp/MoviePilot/assets/115156545/31a3fa25-ae46-4b32-bc55-2dc4a9f8d07d)
![Snipaste_2023-09-22_13-43-23](https://github.com/jxxghp/MoviePilot/assets/115156545/673d9543-dceb-4282-aba2-a37b6f2e4b63)
**标题上看着没有问题，可实际名称识别有的就有问题，然后复制到文档就能看出来这两条标题显示有不一样。
也不知道是php网站问题还是windows问题!**
